scripts/cleanbuild

```diff
@@ -27,7 +28,7 @@ EOW
 [ ! -z "$(git status --porcelain)" ] && echo "$DIRTY_WARNING"
 
 # Fresh clone to ensure our build doesn't rely on anything outside of vcs
-git clone --quiet /src/amazon-ecs-agent /go/src/github.com/aws/amazon-ecs-agent
+git clone -c user.name="cleanbuild" -c user.email="cleanbuild" /src/amazon-ecs-agent /go/src/github.com/aws/amazon-ecs-agent
```

Comment:
What are these values used for? Can you comment on why we need to configure these?

Reply:
`git clone` will try to infer the user name to guess at these values for the initial git repo config. However, when running as a UID in a docker container it is unable to lookup the user name in the passwd file. It panics and exits.

Giving git clone this configuration prevents that error case.